po/fr.po

```diff
@@ -1,66 +1,79 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
 msgid ""
 msgstr ""
+"Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2017-09-04 21:13+0200\n"
+"POT-Creation-Date: 2017-11-02 00:36+0100\n"
+"PO-Revision-Date: 2017-11-02 00:41+0100\n"
+"Language-Team: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.8.7.1\n"
+"Last-Translator: \n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Language: fr\n"
 
 #: ../src/Models/ApplicationProcessModel.vala:68
 msgid "Background Applications"
-msgstr ""
+msgstr "Applications en arrière-plan"
 
 #: ../src/Widgets/Statusbar.vala:25 ../src/Widgets/Statusbar.vala:41
 msgid "CPU:"
-msgstr ""
+msgstr "CPU:"
 
 #: ../src/Widgets/Statusbar.vala:26 ../src/Widgets/Statusbar.vala:34
 msgid "Memory:"
-msgstr ""
+msgstr "Mémoire"
 
 #: ../src/Widgets/Statusbar.vala:27 ../src/Widgets/ProcessView.vala:116
 msgid "GiB"
-msgstr ""
+msgstr "Go"
 
 #: ../src/Widgets/ProcessView.vala:32
 msgid "Process Name"
-msgstr ""
+msgstr "Nom de la tâche"
 
 #: ../src/Widgets/ProcessView.vala:51
 msgid "CPU"
-msgstr ""
+msgstr "CPU"
 
 #: ../src/Widgets/ProcessView.vala:62
 msgid "Memory"
-msgstr ""
+msgstr "Mémoire"
 
 #: ../src/Widgets/ProcessView.vala:72
 msgid "PID"
-msgstr ""
+msgstr "PID"
 
 #: ../src/Widgets/ProcessView.vala:105
 msgid "KiB"
-msgstr ""
+msgstr "Ko"
```

Comment:
Hello! Thanks for translation.
I have a question: why _Ko_ ?
[Binary prefix](https://en.wikipedia.org/wiki/Binary_prefix)

Reply:
Hello!

In French, the translation for "byte" (= 8 bits) is "Octet": a "kilobyte" (KB) is a kilooctet (ko), a megabyte (MB) is a megaoctet (Mo), etc. See https://fr.wikipedia.org/wiki/Octet, for example.

A "bit" en english is also a "bit" in French, though. So kilobit is kbit (or kb) when kilobyte (KiB) is kilooctet (ko).

Frenchies are weirdos :)

Reply:
What about KibiByte? Would it be a KiO ?

Reply:
Yes : Kio. It's seldom used in French: we usually use "ko" for both, although it's not perfectly correct.
See: https://fr.wiktionary.org/wiki/kibioctet

Reply:
Since I'm using here Kibi prefix, a [IEC](https://en.wikipedia.org/wiki/ISO/IEC_80000) standard. In Poland, Russia or Ukraine people also usually say kilo- instead of kibi-. But I decided to use this standard. All translation in Monitor use this prefix. Can You please update Your translation?

Reply:
No problem. Updating...